inyoka_theme_default/templates/portal/usercp/profile.html

```diff
@@ -0,0 +1,35 @@
+{#
+    portal/usercp/settings.html
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    This page shows the user control panel settings page.
+
+    :copyright: (c) 2013-2015 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+
+{% extends 'portal/usercp/overall.html' %}
+
+{% block breadcrumb %}
+  {{ super() }}
+
+  {{ macros.breadcrumb_item(_('Profile'), href('portal', 'usercp', 'profile'))}}
+{% endblock %}
+
+{% set selected = 'profile' %}
+
+{% block content %}
+  {% call macros.outer_form(csrf_token(), form, manually_rendered=True) %}
+    <h4>{% trans %}Avatar settings{% endtrans %}</h4>
+    {% if user.has_avatar %}
+      {{ macros.add_user_avatar(user) }}
+    {% endif %}
+    {{ macros.input(form.avatar) }}
+
+    <h4>{% trans %}Contact addresses{% endtrans %}</h4>
+    {{ macros.inner_form(form, ['email', 'gpgkey', 'jabber', 'icq', 'msn', 'aim', 'yim', 'skype', 'wengophone', 'sip']) }}
```

Comment:
as [mentioned](https://github.com/inyokaproject/theme-default/pull/105#issuecomment-107076464) checkboxes to (un-)hide email and jabber in the profile are missing